app/uk/gov/hmrc/pensionschemereturn/auth/PsrAuth.scala

```diff
@@ -19,10 +19,10 @@ package uk.gov.hmrc.pensionschemereturn.auth
 import play.api.Logging
 import play.api.mvc.{Request, Result}
 import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals
-import uk.gov.hmrc.auth.core.retrieve.{~, Name}
+import uk.gov.hmrc.auth.core.retrieve.{Name, ~}
```

Comment:
When I ran sbt compile, this line (and 5 more files) got reverted back how they were before scalafix. I.e. from `{Name, ~}` to `{~, Name}`.
If scalafix and compile produce a different formatting, should we always use scalafix after of compile?
Otherwise people will accidentally revert back at some point....

Reply:
Set it so that it auto runs on compile - we won't need to explicitly run it now